DevoxxClientMobile/src/main/java/com/devoxx/util/DevoxxNotifications.java

```diff
@@ -150,21 +150,14 @@ public void preloadScheduledSessions() {
      */
     public void preloadingScheduledSessionsDone() {
```

Comment:
I have been thinking if we should get rid of this method and directly add notification to `LocalNotificationService` in `addAlreadyScheduledSessionNotifications()`.